experiment/references.md

```diff
@@ -1 +1,3 @@
-### Link your references in here
\ No newline at end of file
+- [https://ocw.mit.edu/NR/rdonlyres/Aeronautics-and-Astronautics/16-20Structural-%20MechanicsFall2002/7E90098F-C973-4B10-9C91-94730623C3B9/0/unit22.pdf](http://ocw.mit.edu/NR/rdonlyres/Aeronautics-and-Astronautics/16-20Structural-%20MechanicsFall2002/7E90098F-C973-4B10-9C91-94730623C3B9/0/unit22.pdf)
+- [https://www.aerostudents.com/files/vibrations/multipleDegreeOfFreedomSystems.pdf](http://www.aerostudents.com/files/vibrations/multipleDegreeOfFreedomSystems.pdf)
+- "Earthquake Resistant Design of Structures" by Pankaj Agarwal, Manish Shrikande
```

Comment:
Please use https links only. Also do not use link in the description of the reference. Please use a descriptive phrase/sentence.